docs/sources/write/reuse-content/_index.md

```diff
@@ -1,20 +1,90 @@
 ---
-title: Reuse content
-menuTitle: Reuse
-description: Learn about reusing content in your documentation.
-weight: 800
 aliases:
   - /docs/writers-toolkit/write/reuse-content
-Keywords:
+date: 2022-02-22
+description: Learn about reusing content in your documentation.
+keywords:
   - sharing
   - reuse content
+menuTitle: Reuse
+title: Reuse content
+weight: 800
 ---
 
 # Reuse content
 
 Grafana's publication process can handle reusing content from snippets to whole directories.
 
 There are some limited cases where it may be useful to reuse content in more than one location.
-Let's say that a procedure for adding a panel to a dashboard applies to an open source and enterprise product.
+For example, a procedure for adding a panel to a dashboard applies to an open source and enterprise product.
+
+## Before you begin
+
+Before you begin reusing content, it's important to understand the tradeoffs.
+Reusing content introduces maintenance overheads that can be greater than the overhead of maintaining separate content.
+
+You can reuse content in the following ways:
+
+1. Link to definitive content
+1. Share chunks of content
+1. Mount directories of content
+
+The following sections discuss each option in detail.
+
+### Link to definitive content
+
+Instead of including content directly in your documentation, you can link to the definitive source instead.
+
+#### Pros
+
+- You can reuse content immediately.
+
+#### Cons
+
+- You have no control over the content.
+
+  If the definitive content changes, your link may need to be updated.
+
+### Share chunks of content
+
+You can reuse chunks of content using the `docs/shared` shortcode.
+To do this, refer to [Reuse shared content](https://grafana.com/docs/writers-toolkit/write/reuse-content/reuse-shared-content/).
+
+#### Pros
+
+- You can reuse small amounts of content.
+- You can maintain the content in one place.
+- You can compose the shared content with other content.
+
+#### Cons
+
+- You need to perform the initial setup to create the shared content.
+
+### Mount directories of content
+
+You can reuse whole directories of content using Hugo mounts.
+To do this, refer to [Reuse content with Hugo mounts](https://grafana.com/docs/writers-toolkit/write/reuse-content/hugo-mounts).
+
+With this option, you share not only content but also structure: each page has exactly the same heading structure.
```

Comment:
```suggestion
With this option, you share content and structure. Each page has exactly the same heading structure.
```

We could use a semicolon here... but I usually try to split it into two distinct sentences instead.  Sentences can be short.

Reply:
I'm always fine with short sentences so the linter doesn't ask me to think about if my use of a semi-colon is justified or not :D